<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="userInfoTab">
       <attribute name="title">
        <string>User</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="passChangeTab">
       <attribute name="title">
        <string>Change password</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>10</y>
          <width>381</width>
          <height>251</height>
         </rect>
        </property>
        <property name="title">
         <string>Password change</string>
        </property>
        <widget class="QWidget" name="">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>30</y>
           <width>285</width>
           <height>166</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <widget class="QLabel" name="oldPassLabel">
              <property name="text">
               <string>Old password :</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="oldPassLE"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="newPassLabel">
              <property name="text">
               <string>New password :</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="newPassLE"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="confirmPassLabel">
              <property name="text">
               <string>Confirm password :</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="confirmPassLE"/>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="changePassBtn">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Change password</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
